### ISSUE TITLE
cuda error with SequenceLast.

### ISSUE THREAD

**Issue by yajiedesign** [apache/incubator-mxnet#17888]
## Description
 CUDA: an illegal memory access was encountered error with SequenceLast.

### Error Message
 CUDA: an illegal memory access was encountered 

## To Reproduce
simple lstm net work.
`class LSTMNetwork(gluon.HybridBlock):
    def __init__(self, ctx=mx.gpu(0), **kwargs):
        super(LSTMNetwork, self).__init__(**kwargs)
        self.ctx = ctx
        with self.name_scope():
            self.main = gluon.nn.HybridSequential()
            self.main.add(nn.Dropout(0.5))
            self.main.add(rnn.LSTM (128, num_layers=1, dropout=0, bidirectional=True))
            self.main.add(nn.BatchNorm())
            self.output = gluon.nn.HybridSequential()
            self.output.add(nn.Dropout(0.5))
            self.output.add(nn.Dense(2, flatten=True))

    def hybrid_forward(self, F, x, **kwargs):
        x = x.transpose((2, 0, 1))
        output = self.main(x)
		# this is work done
        # output1 =  output[-1]
        output = F.SequenceLast(output)
        output = self.output(output)
        return output
`

## Environment
----------Python Info----------
Version      : 3.7.3
Compiler     : MSC v.1915 64 bit (AMD64)
Build        : ('default', 'Mar 27 2019 17:13:21')
Arch         : ('64bit', 'WindowsPE')
------------Pip Info-----------
Version      : 20.0.2
Directory    : D:\Anaconda3\lib\site-packages\pip
----------MXNet Info-----------
Version      : 1.6.0
Directory    : D:\Anaconda3\lib\site-packages\mxnet
Num GPUs     : 1
Hashtag not found. Not installed from pre-built package.
----------System Info----------
Platform     : Windows-10-10.0.18362-SP0
system       : Windows
node         : DESKTOP-T6GH9E6
release      : 10
version      : 10.0.18362
----------Hardware Info----------
machine      : AMD64
processor    : Intel64 Family 6 Model 158 Stepping 12, GenuineIntel
Name
Intel(R) Core(TM) i7-9700KF CPU @ 3.60GHz

----------Network Test----------
Setting timeout: 10
Timing for MXNet: https://github.com/apache/incubator-mxnet, DNS: 0.0040 sec, LOAD: 1.4730 sec.
Timing for GluonNLP GitHub: https://github.com/dmlc/gluon-nlp, DNS: 0.0000 sec, LOAD: 1.4230 sec.
Timing for GluonNLP: http://gluon-nlp.mxnet.io, DNS: 0.3570 sec, LOAD: 2.3780 sec.
Timing for D2L: http://d2l.ai, DNS: 0.5670 sec, LOAD: 0.8950 sec.
Timing for D2L (zh-cn): http://zh.d2l.ai, DNS: 0.0040 sec, LOAD: 0.3690 sec.
Timing for FashionMNIST: https://repo.mxnet.io/gluon/dataset/fashion-mnist/train-labels-idx1-ubyte.gz, DNS: 1.0500 sec, LOAD: 2.0910 sec.
Timing for PYPI: https://pypi.python.org/pypi/pip, DNS: 0.0030 sec, LOAD: 14.5190 sec.
Timing for Conda: https://repo.continuum.io/pkgs/free/, DNS: 0.0040 sec, LOAD: 1.6770 sec.
